monitoring/mock_uss/msgsigning/database.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import Dict, Optional
+
+from monitoring.monitorlib.multiprocessing import SynchronizedValue
+from implicitdict import ImplicitDict
+
+# TODO Use this database to dynamically alter which key pair to use for message signing activities.
+
+
+class Database(ImplicitDict):
+    """Simple in-memory pseudo-database tracking whether or not to use a valid key pair for message signing activities."""
+
+    use_valid_keypair: bool = True
```

Comment:
This will work, but I'd suggest that maybe we could allow for much more flexibility without almost any additional effort:

```suggestion
    public_key_name: str = "messagesigning/mock_faa_priv.der"
    private_key_name: str = "messagesigning/mock_faa_priv.pem"
```

With this, a future endpoint to change the keypair could accept any keypair available to mock_uss without additional code, and the file names could be fully resolved without any additional hardcoding.